docs/developer/app-store/apps/cms.mdx

```diff
@@ -139,7 +139,7 @@ Each of the CMS platforms has slight differences in configuration and behavior d
 ### Contentful
 
 - Fields are autocompleted after you provide [space ID](https://www.contentful.com/help/find-space-id/) and a [token](https://www.contentful.com/help/personal-access-tokens/).
-- Entries will be created with an ID that matches Saleor product variant ID.
+- Entries will be created with ID autogenerated by Contentful. You should set the variant ID field to be unique.
```

Comment:
**thought**: it wouldn't be clear to me (from reading this point) how these two sentences relate to each other

Reply:
I will revisit this another time, because the same sentence is in every app. I'd like to merge this asap because app is released already